src/components/seo.tsx

```diff
@@ -40,7 +40,7 @@ export const SEO: React.FC<SEOProps> = ({
   return (
     <Head>
       <title>{seo.title}</title>
-      <link rel="icon" href="/favicon.ico" />
+      <link rel="icon" href="/bjerk-favicon.svg" />
```

Comment:
Should probably be named favicon to adhere to standards

Reply:
Favicon is just the name of the type of icon that is used in the tab, but as it is Bjerk's favicon - imo it's a fitting name.

Reply:
If the metadata reference is not retrieved, the browser looks for a favicon (placed in the root directory, named "favicon")

Not sure if it works with SVG though.

Reply:
Ah, okay.